userprefs.cgi

```diff
@@ -72,6 +72,10 @@ sub DoAccount {
       }
```

Comment:
For clarification: the changes in this file handle the comments in [this section](https://github.com/mozmeao/consent-banner?tab=readme-ov-file#how-can-visitors-change-their-cookie-settings-after-interacting-with-the-banner) of the consent-banner README, correct? Just want to make sure that was properly addressed.

Reply:
I made some changes to follow the requirements. Including:

- A separate non-authenticated cookie settings page
- Links to the cookie settings page in easy to find places.